includes/class-freemius.php

```diff
@@ -13076,15 +13346,16 @@ function activate_migrated_license(
             $license_key,
             $is_marketing_allowed = null,
             $plugin_id = null,
+            $sites = array(),
             $blog_id = null
         ) {
             $this->_logger->entrance();
 
             $result = $this->activate_license(
                 $license_key,
-                $this->is_network_active() ?
+                ( empty( $sites ) && is_null( $blog_id ) && $this->is_network_active() ) ?

```

Comment:
@fajardoleo the previous logic was not validating that the `$blog_id` is null, meaning, it may not be backward compatible. Are you confident about this change?

Reply:
@vovafeldman `$blog_id` is new to this branch, so it's always `null` in old usages.